team-manager/src/test/intergration/java/ua/knu/knudev/teammanager/service/AccountProfileServiceIntegrationTest.java

```diff
@@ -0,0 +1,231 @@
+//package ua.knu.knudev.teammanager.service;
+//
+//import jakarta.transaction.Transactional;
+//import org.junit.jupiter.api.*;
+//import org.junit.jupiter.api.extension.ExtendWith;
+//import org.mockito.*;
+//import org.mockito.junit.jupiter.MockitoExtension;
+//import org.springframework.beans.factory.annotation.Autowired;
+//import org.springframework.boot.test.context.SpringBootTest;
+//import org.springframework.boot.test.mock.mockito.MockBean;
+//import org.springframework.context.annotation.Import;
+//import org.springframework.mock.web.MockMultipartFile;
+//import org.springframework.test.context.ActiveProfiles;
+//import org.springframework.test.context.ContextConfiguration;
+//import org.springframework.test.context.junit.jupiter.SpringExtension;
+//import ua.knu.knudev.fileserviceapi.api.FileServiceApi;
+//import ua.knu.knudev.knudevsecurityapi.api.AccountAuthServiceApi;
+//import ua.knu.knudev.knudevsecurityapi.request.AccountCreationRequest;
+//import ua.knu.knudev.knudevsecurityapi.response.AccountCreationResponse;
+//import ua.knu.knudev.knudevsecurityapi.constant.AccountRole;
+//import ua.knu.knudev.teammanager.config.TeamManagerConfig;
+//import ua.knu.knudev.teammanager.config.TestConfig;
+//import ua.knu.knudev.teammanager.domain.AccountProfile;
+//import ua.knu.knudev.teammanager.domain.Department;
+//import ua.knu.knudev.teammanager.domain.Specialty;
+//import ua.knu.knudev.teammanager.mapper.AccountProfileMapper;
+//import ua.knu.knudev.teammanager.repository.AccountProfileRepository;
+//import ua.knu.knudev.teammanagerapi.dto.AcademicUnitsIds;
+//import ua.knu.knudev.teammanagerapi.exception.AccountException;
+//import ua.knu.knudev.teammanagerapi.response.AccountRegistrationResponse;
+//
+//import java.util.Set;
+//import java.util.UUID;
+//
+//import static org.mockito.ArgumentMatchers.any;
+//import static org.mockito.Mockito.*;
+//import static org.junit.jupiter.api.Assertions.*;
+//
```

Comment:
let`s delete it

Reply:
this is for my next pr. I intially wanted to make intergration tests here, but its already too big pr without them